forms/aws/secrets.tf

```diff
@@ -13,6 +13,19 @@ resource "aws_secretsmanager_secret_version" "notify_api_key" {
   secret_string = var.ecs_secret_notify_api_key
 }
 
+# Ignore using global encryption key
+#tfsec:ignore:AWS095
+resource "aws_secretsmanager_secret" "list_manager_api_key" {
+  name                    = "list_manager_api_key"
+  recovery_window_in_days = 0
+}
+
+resource "aws_secretsmanager_secret_version" "list_manager_api_key" {
+  secret_id     = aws_secretsmanager_secret.list_manager_api_key.id
+  secret_string = var.ecs_list_manager_api_key
```

Comment:
for consistency we usually prefix variables that are secret with `ecs_secret_` so there's no confusion.